test/persistence-hooks.suite.js

```diff
@@ -3267,13 +3256,8 @@ module.exports = function(dataSource, should, connectorCapabilities) {
                 id: existingInstance.id,
                 name: 'updated name',
               },
+              isNewInstance: false,
             });
-            // For non-atomic implementation of upsertWithWhere on update, it calls
-            // updateAttributes. loaded hook of updateAttributes does not provide
-            // isNewInstance.
-            if (dataSource.connector.upsertWithWhere) {
-              expectedContext.isNewInstance = false;
-            }
```

Comment:
Wow Good catch @bajtos 👍  💯  Thanks for reminding; I was definitely forgetting this!